### DISCUSSION TITLE
Error with Join Slack Page

### DISCUSSION THREAD

**Discussion by ReedAndreas** [openfoodfacts/openfoodfacts-server#8248] [Q&A] [answered]
Hi all! I am looking to join the slack at https://slack.openfoodfacts.org/ but am receiving a 504 timeout error on nginx. Any insight on how I can get on the slack given this problem?

**Comment by ReedAndreas** [comment, marked as answer]
This issue seems to be resolved now!

**Reply by alexgarel** [reply]
yes, see https://github.com/openfoodfacts/openfoodfacts-infrastructure/issues/204
